www/~/%username/emails/verify.html.spt

```diff
@@ -12,15 +12,11 @@ from gratipay.models.participant import email
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    if 'email' in request.qs:
-
-        # Deprecated in GH-3965. We shouldn't be generating any more of these,
-        # and this can be removed when we're confident that there are no more
-        # old-style verification links hanging out there.
-
-        email_address = request.qs['email']
+    if 'email2' in request.qs:
```

Comment:
Let's add a comment similar to the one we're removing to explain what's going on here.

Reply:
@whit537 Done.